content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -64,6 +64,7 @@ The [latest version of the .NET Tracer][4] can automatically instrument the foll
 | Aerospike                       | `Aerospike.Client` 4.0.0+                                                                            | `Aerospike`          |
 | ASP.NET Core                    | `Microsoft.AspNetCore`</br>`Microsoft.AspNetCore.App`</br>2.0+ and 3.0+                              | `AspNetCore`         |
 | Azure Functions                 | `Microsoft.Azure.Webjobs` 3.0+                                                                       | `AzureFunctions`     |
+| AWS Kinesis                     | `AWSSDK.Kinesis`  3.0+                                                                               | `AwsKinesis`         |
```

Comment:
```suggestion
| Amazon Kinesis                     | `AWSSDK.Kinesis`  3.0+                                                                               | `AwsKinesis`         |
```

Reply:
Should I change every other service then? I was just keeping consistency with the current docs.

Reply:
@duncanista thanks for following up, and no worries about other services. There's a separate docs card (DOCS-6255) open to review AWS service names throughout the documentation which should take care of those.

Reply:
Thanks for the heads up! Committing suggestion!